tasks/install.yml

```diff
@@ -0,0 +1,289 @@
+---
+- import_tasks: apt_install.yml
+  when: ansible_pkg_mgr == "apt"
+
+- name: Add gitlab user
+  user:
+      name: git
+      shell: /bin/nologin
+      comment: 'GitLab'
+      state: present
+      groups: redis
+
+- name: Install bundler gem
+  gem:
+    name: bundler
+  become: yes
+  become_user: git
+
+- import_tasks: postgres_install.yml
+
+- name: Creates redis socket directory
+  file:
+    path: "{{redis_unixsocket | dirname }}"
+    state: directory
+    owner: redis
+    group: redis
+    mode: 0755
+
+- name: Clone gitlab to {{gitlab_install_path | mandatory}}
+  git:
+    depth: 1
+    repo: https://gitlab.com/gitlab-org/gitlab-ee.git
+    dest: "{{ gitlab_install_path | mandatory }}"
+    version: "{{ gitlab_version_number | mandatory }}"
+    force: yes
+  when: gitlab_version == "ee"
+
+- name: Clone gitlab to {{gitlab_install_path}}
+  git:
+    depth: 1
+    repo: https://gitlab.com/gitlab-org/gitlab-ce.git
+    dest: "{{ gitlab_install_path | mandatory}}"
+    version: "{{ gitlab_version_number }}"
```

Comment:
one time it's mandatory, one time it's not 🤔 This is one disadvantage of having duplicate code.

Reply:
fixed